examples/cephfs/storageclass.yaml

```diff
@@ -17,7 +17,7 @@ parameters:
   # CephFS filesystem name into which the volume shall be created
   fsName: myfs
 
-  # Ceph pool into which the volume shall be created
+  # Ceph pool into which the volume shall be created (this is an optional parameter)
```

Comment:
Reword: "(optional) Ceph pool into which volume data shall be stored (also change the string in deploy-cephfs.md)

Also comment the line below (line 21)